oem/ibm/requester/dbus_to_file_handler.cpp

```diff
@@ -236,6 +236,49 @@ void DbusToFileHandler::newCsrFileAvailable(const std::string& csr,
                                PLDM_FILE_TYPE_CERT_SIGNING_REQUEST);
 }
 
+void DbusToFileHandler::newChapDataFileAvailable(
+    const std::string& chapNameStr, const std::string& chapPasswordStr)
```

Comment:
Pls change "password" to "user challenge"

Reply:
done